javascript/javascript1/week3/smartEase.js

```diff
@@ -0,0 +1,65 @@
+//SAVE A NOTE
+const notes = [];
+
+
+function saveNote(content, id) {
+    const addNotes = { content: content, id: id }
+    notes.push(addNotes);
```

Comment:
Variables (and constants) are normally named like objects, for example `note`. The name `addNotes` makes it sound like a function that will do something. A name like `note` makes it sound like what it is, an object that holds information.
```suggestion
    const note = { content: content, id: id }
    notes.push(note);
```

Reply:
Yes...I will keep that in mind the next  and will change it now